helios-services/src/main/java/com/spotify/helios/servicescommon/ZooKeeperRegistrarUtil.java

```diff
@@ -0,0 +1,331 @@
+/*
```

Comment:
At first it looked like `ZooKeeperRegistrar` was a good class to put this, but I think that class does something very different and mostly used as an instance whereas registerHost() and deregisterHost() work nicely as static methods.